### DISCUSSION TITLE
Figures to images

### DISCUSSION THREAD

**Discussion by DMuzer** [NetTopologySuite/NetTopologySuite#634] [Q&A]
Hello Everyone! Is there any function to get image of figures? I mean to have png or svg, something like it possible to get from shapely in python?

**Comment by DGuidi** [comment]
no asbuilt function to achieve this. 
you can howewer get retrieve easily the wkt or geojson representation of your data and use an online service, like https://arthur-e.github.io/Wicket/sandbox-gmaps3.html (or similar) or https://geojson.io/ (if your data is in EPSG:4326)

**Comment by FObermaier** [comment]
Depending on the platform you use [NetTopologySuite.Windows](https://github.com/NetTopologySuite/NetTopologySuite.Windows) may be your solution.   
It should be straight forward to code sth. like this for dotnet.MAUI or SkiaSharp.
